persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -418,6 +418,10 @@ insertOrIgnoreMessage:
 INSERT OR IGNORE INTO Message(id, content_type, conversation_id, date, sender_user_id, sender_client_id, status, visibility, expects_read_confirmation)
 VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?);
 
+insertOrIgnoreBullkSystemMessage:
+INSERT OR IGNORE INTO Message(id, content_type, conversation_id, date, sender_user_id, sender_client_id, status, visibility, expects_read_confirmation)
```

Comment:
Should maybe limit the insertion to only 1:1 conversations and groups?

Reply:
Hmm - ok! I'll do it but still no harm of adding the message for all.